bin/pytorch_inference/.gitignore

```diff
@@ -0,0 +1,3 @@
+input_file
```

Comment:
If `evaluate.py` runs successfully these files should be cleaned up. 

I'll raise a change to always cleanup these files as the actual names are arguments to the script anyway. Happy to merge this as is I will revert this file when I make the PR